### PR TITLE
fix: git blame and file edit render with correct route

### PR DESCRIPTION
**Pull request by shaurya-harness**
fix: git blame and file edit render with correct route

PS: File Editor component changes in this PR is the old component and @3em is working on moving it to components-v2, the current PR only adds changes to make it render so that he can see the old component and how the route is working

### CODE REVIEW

File: apps/gitness/src/components/FileEditor.tsx [hunk 1]
```diff
@@ -153,158 +153,156 @@ export const FileEditor: React.FC = () => {
...
+                  onFocus={({ target }) => {
+                    const value = (parentPath ? parentPath + FILE_SEPERATOR : '') + fileName
+                    setFileName(value)
+                    setParentPath('')
+                    setTimeout(() => {
+                      target.setSelectionRange(value.length, value.length)
+                      target.scrollLeft = Number.MAX_SAFE_INTEGER
+                    }, 0)
+                  }}
+                  placeholder="Name your file..."
```
Comment (cjlee01) [inline apps/gitness/src/components/FileEditor.tsx:235]: should this be added to the views.json?

File: apps/gitness/src/components/FileEditor.tsx [hunk 2]
```diff
@@ -153,158 +153,156 @@ export const FileEditor: React.FC = () => {
...
+              className={'rounded-lg border border-primary/10 bg-primary-foreground p-0.5'}
+            >
+              <ToggleGroupItem
+                value={'contents'}
+                className="h-7 rounded-md border border-transparent text-xs font-medium disabled:opacity-100 data-[state=on]:border-primary/10"
+              >
+                Contents
+              </ToggleGroupItem>
+              <ToggleGroupItem
+                value={'changes'}
```
Comment (knagurski) [inline apps/gitness/src/components/FileEditor.tsx:276]: No need for parens
```suggestion
              unselectable="on"
              className="rounded-lg border border-primary/10 bg-primary-foreground p-0.5"
            >
              <ToggleGroupItem
                value="contents"
                className="h-7 rounded-md border border-transparent text-xs font-medium disabled:opacity-100 data-[state=on]:border-primary/10"
              >
                Contents
              </ToggleGroupItem>
              <ToggleGroupItem
                value="changes"
```

File: apps/gitness/src/components/FileEditor.tsx [hunk 3]
```diff
@@ -153,158 +153,156 @@ export const FileEditor: React.FC = () => {
...
+          </ListActions.Right>
+        </ListActions.Root>
+        <Spacer size={5} />
+        <StackedList.Root onlyTopRounded borderBackground>
+          <StackedList.Item disableHover isHeader className="px-3 py-2.5">
+            <ToggleGroup
+              onValueChange={(value: ViewTypeValue) => {
+                if (value) {
+                  setView(value)
+                }
```
Comment (knagurski) [inline apps/gitness/src/components/FileEditor.tsx:262]: Can this be something falsey? Can you just call it?
```suggestion
              onValueChange={(value: ViewTypeValue) => setView(value)}
```

or even
```suggestion
              onValueChange={setView}
```

File: apps/gitness/src/components/FileEditor.tsx [hunk 4]
```diff
@@ -153,158 +153,156 @@ export const FileEditor: React.FC = () => {
...
-          )}
-        </SandboxLayout.Content>
-      </SandboxLayout.Main>
+              value={view}
+              type="single"
+              unselectable={'on'}
+              className={'rounded-lg border border-primary/10 bg-primary-foreground p-0.5'}
+            >
+              <ToggleGroupItem
+                value={'contents'}
```
Comment (knagurski) [inline apps/gitness/src/components/FileEditor.tsx:270]: A few more
```suggestion
              unselectable="on"
              className="rounded-lg border border-primary/10 bg-primary-foreground p-0.5"
            >
              <ToggleGroupItem
                value="contents"
```

File: apps/gitness/src/components/FileEditor.tsx (hunk 1 above)
Comment (shaurya-harness) [reply apps/gitness/src/components/FileEditor.tsx:235]: can be taken care of after pixel point team moves it to components-v2

File: apps/gitness/src/components/FileEditor.tsx [hunk 5]
```diff
@@ -153,158 +153,156 @@ export const FileEditor: React.FC = () => {
...
+                    }, 0)
+                  }}
+                  placeholder="Name your file..."
+                />
+              </BreadcrumbItem>
+            </ButtonGroup.Root>
+          </ListActions.Left>
+          <ListActions.Right>
+            <Button variant="default" size="sm" onClick={() => setIsCommitDialogOpen(true)}>
+              Commit Changes
```
[the remark this answers is not in the document]
Comment (shaurya-harness) [reply apps/gitness/src/components/FileEditor.tsx:242]: can be taken care of after pixel point team moves it to components-v2

**Review by cjlee01 — APPROVED** [review]

**Pull request merged**